src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -373,9 +373,6 @@ private set
 #if FEATURE_UTF32
                         || value.Equals(Encoding.UTF32)
 #endif // FEATURE_UTF32
-#if FEATURE_UTF7
-                        || value.Equals(Encoding.UTF7)
-#endif // FEATURE_UTF7
```

Comment:
Was this already dead code because we weren't setting FEATURE_UTF7?  Are we setting FEATURE_UTF32?

Reply:
Correct, neither of these is set by the build process. Would you prefer I remove both?

Reply:
If it's dead code, yeah.

Reply:
I looked at the generated IL from System.IO.Compression and seeing neither UTF7 nor UTF32 is defined. either we need to confirm these are intentional not including them and then remove the code for UTF32 too, or we enable such ifdef's (not enabling UTF7 though)



 private set
 {
  if (value != null && (value.Equals(Encoding.BigEndianUnicode) || value.Equals(Encoding.Unicode)))
  {
   throw new ArgumentException(SR.EntryNameEncodingNotSupported, "EntryNameEncoding");
  }
  _entryNameEncoding = value;
 }

---
In reply to: [436265058](https://github.com/dotnet/runtime/pull/37535#discussion_r436265058) [](ancestors = 436265058)